### PR TITLE
Gidk 3dc1z6 show nav prod

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/vat-report/vatReport.component.html [hunk 1]
```diff
@@ -13,226 +16,195 @@
                                     <div class="tax-heading col-md-3 col-sm-3 col-12">
                                         <label>{{commonLocaleData?.entity}}:</label>
                                     </div>
-                                    <div class="tax-dropdown col-md-9 col-sm-9 col-12 linked-entities">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/vat-report/vatReport.component.html:16]: @shalinee-walkover please check this code is getting removed

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/vat-report/vatReport.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/vat-report/vatReport.component.html]: added, Done

File: apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.ts [hunk 2]
```diff
@@ -282,4 +300,13 @@ export class VatReportTransactionsComponent implements OnInit, OnDestroy {
     public navigateToPreviousPage(): void {
         this.router.navigate(['/pages/vat-report'], { state: { taxNumber: this.vatReportTransactionsRequest.taxNumber, from: this.vatReportTransactionsRequest.from, to: this.vatReportTransactionsRequest.to } })
     }
+
+    /**
+     * Handles GST Sidebar Navigation
+     *
+     * @memberof VatReportTransactionsComponent
+     */
+     public handleNavigation(): void {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.ts:309]: Need to remove this

File: apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.html [hunk 3]
```diff
@@ -1,7 +1,10 @@
 <ng-container appTranslate [file]="'vat-report'" (localeData)="localeData = $event"
     (commonLocaleData)="commonLocaleData = $event">
     <div class="branch-transfer-report mt-2">
-        <hamburger-menu [pageHeading]="localeData?.tab_heading"></hamburger-menu>
+        <hamburger-menu [allowSidebarToggle]="false" [pageHeading]="localeData?.tab_heading"></hamburger-menu>
+        <div class="help-support-aside setting-sidebar-wrapper gst-sidebar-wrapper" *ngIf="asideGstSidebarMenuState === 'in'">
+            <tax-sidebar (navigateEvent)="handleNavigation()" (clickOutside)="isMobileScreen ? asideGstSidebarMenuState = 'out' : ''" [exclude]="'#primary-new-header'" class="gst-sidebar"></tax-sidebar>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.html:6]:
```suggestion
            <tax-sidebar (clickOutside)="isMobileScreen ? asideGstSidebarMenuState = 'out' : ''" [exclude]="'#primary-new-header'" class="gst-sidebar"></tax-sidebar>
```

**Comment by shalinee-walkover** [comment]
https://app.clickup.com/t/3dc1z6

**Review by prateek-giddh — APPROVED** [review]

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.html]: Done

File: apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.ts (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/vat-report/transactions/vatReportTransactions.component.ts]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**